### PR TITLE
Scripts to Rule Them All (PHP built-in web server support)

### PR DESCRIPTION
**Pull request by jakefolio**
This PR adds the Scripts to Rule Them All pattern, https://github.com/github/scripts-to-rule-them-all,  to allow for a more simplified development experience. Also, I've included a server.php router file for PHP built-in web server. I have re-organized the README to move local development higher up and created an entire section for Scripts to Rule Them All.

Related to #262

### CODE REVIEW

File: server.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+define('WEB_PATH', __DIR__ . '/web/');
+
+if (preg_match('/\.css|\.js|\.jpg|\.png|\.map|\.svg|\.ico$/', $_SERVER['REQUEST_URI'], $match)) {
+    $mimeTypes = [
```
Comment (vlucas) [inline server.php]: I think you can just `return false;` to serve static assets as-is, right?

**Comment by jakefolio** [comment]
@vlucas the update isn't too simplistic is it? I tested locally and it responded correctly, correct headers too.

**Comment by vlucas** [comment]
Not as long as you don't plan on hitting a `.php` file directly :)

**Comment by jakefolio** [comment]
It should be locked down to the web directory, so I didn't really feel it would be necessary to filter out .php file requests, but can add that in.

**Comment by vlucas** [comment]
Yeah, given that no .php files are under `web`, you honestly should be fine.

**Comment by chartjes** [comment]
Thanks @jakefolio

**Pull request merged**